helics/capi.py

```diff
@@ -717,9 +718,33 @@ class HelicsFederateState(IntEnum):
 helics_state_pending_finalize = HelicsFederateState.PENDING_FINALIZE
 
 
+def generate_cleanup_callback(obj):
+
+    if issubclass(obj.__class__, HelicsFederate):
```

Comment:
Less code to use `isinstance` instead of `issubclass`, e.g. `isinstance(obj, HelicsFederate)`

Reply:
Resolved in https://github.com/GMLC-TDC/pyhelics/pull/26/commits/c226212d2d6cac119f85d477ce360e626e8ccbd3.